packages/create-bison-app/template/_templates/graphql/new/graphql.ejs

```diff
@@ -3,7 +3,7 @@ to: graphql/modules/<%= name %>.ts
 ---
 <% camelized = h.inflection.camelize(name) -%>
 <% plural = h.inflection.pluralize(camelized) -%>
-import { objectType, extendType, inputObjectType, arg, nonNull /*, enumType*/ } from 'nexus';
+import { objectType, extendType, inputObjectType, queryField, mutationField, arg, list, nonNull /*, enumType*/ } from 'nexus';
```

Comment:
I tested this in a new bison app, and this line has lint errors (too long and unused import). This is also an issue in the user module file.

Reply:
🤦🏼 Forgot I refactored `list` back out. 
I gotta get in the habit of running the linter check. 

Thanks

Reply:
927e666 🤦🏼